contracts/actions/post/collect/SimpleCollectAction.sol

```diff
@@ -45,13 +49,12 @@ contract SimpleCollectAction is ISimpleCollectAction, OwnableMetadataBasedPostAc
     bytes32 constant PARAM__END_TIMESTAMP = 0xe2a4a768f409ba480a321a7d36ec9da16e9eae60a25bb0aeccf334822cc859a8;
     /// @custom:keccak lens.param.recipients
     bytes32 constant PARAM__RECIPIENTS = 0x7f7e01c87d5278dd08505253491cf5d6b30930036f6afa2ae22a980882f2cac1;
-    /// @custom:keccak lens.param.referralFee
-    bytes32 constant PARAM__REFERRAL_FEE = 0x6dff2c1710f2154b19d8cf5d6f7d8f5b3909222c3cdd8801486403e4d423b1b6;
+    /// @custom:keccak lens.param.referralFeeBps
+    bytes32 constant PARAM__REFERRAL_FEE_BPS = 0x0528211c8ce09d8b4bbf47978d7c2b7901461b28da5f4da81efb3058169ea470;
```

Comment:
hmm breaking change with knowing the key

Reply:
OK, after we talked, we will keep the previous key